src/Tendermint/Serialization.cpp

```diff
@@ -0,0 +1,120 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Serialization.h"
+
+#include "../Tendermint/Address.h"
+#include "../Base64.h"
+#include <TrustWalletCore/TWHRP.h>
+
+using namespace TW;
+using namespace TW::Cosmos;
+using namespace TW::Cosmos::Proto;
+
+using json = nlohmann::json;
+
+const std::string AMINO_PREFIX_SEND_MESSAGE = "cosmos-sdk/MsgSend";
+const std::string AMINO_PREFIX_TRANSACTION = "auth/StdTx";
+const std::string AMINO_PREFIX_PUBLIC_KEY = "tendermint/PubKeySecp256k1";
```

Comment:
use `static` please.

Reply:
> use `static` please.

done in 50fc651d6e7f5798ecc5b2baae695436fc3b6cfc